lib/can/CANBuffer.cpp

```diff
@@ -44,11 +47,11 @@ uint32_t CANBuffer::getFlags() {
     return m_eventFlags.get();
```

Comment:
looks like you set the full flag but never actually act on it. should it clear old commands? i think at the very least the irq should check if !full before bothering to even read the can msg

Reply:
It's a circular buffer. It already clears old commands. Its just an option for the implementor for error messaging etc. Which I plan to add later on

Reply:
It shouldn't check if full because it should overwrite

Reply:
Is there a reason u set the flag then?

Reply:
Its just an option for the implementor for error messaging etc. Which I plan to add later on